Installer/source/main.cpp

```diff
@@ -1577,6 +1746,14 @@ int main(int argc, char **argv)
 	copy_app[33] = '1';
 	gprintf("%s &\r\n%s", original_app, copy_app);
 
+	// For vWii use additional content containing priiloader, which replaces content 1
+	if (is_vwii)
+	{
+		memset(additional_app,0,64);
+		sprintf(additional_app, "/title/%08x/%08x/content/%08x.app",TITLE_UPPER(title_id),TITLE_LOWER(title_id),id);

```

Comment:
would it be faster if you were to do a strncpy of the copy_app instead of the sprintf again?
its the same value except for the changed 2 anyway